src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_RADIUS = 10.0;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Color defaultColor = Color.WHITE;
```

Comment:
lets make it proper constant
```suggestion
    private static final Color DEFAULT_COLOR = Color.WHITE;
```